libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -58,16 +58,23 @@ interface VegaDialogProps {
  * Default dialog content
  */
 export const VegaDialog = ({ transaction }: VegaDialogProps) => {
-  const { VEGA_EXPLORER_URL } = useEnvironment();
+  const { VEGA_EXPLORER_URL, VEGA_ENV } = useEnvironment();
 
   let content = null;
   if (transaction.status === VegaTxStatus.Requested) {
     content = (
-      <p data-testid={transaction.status}>
-        {t(
-          'Please open your wallet application and confirm or reject the transaction'
+      <>
+        <p data-testid={transaction.status}>
+          {t(
+            'Please open your wallet application and confirm or reject the transaction'
+          )}
+        </p>
+        {VEGA_ENV !== Networks.MAINNET && (
+          <p data-testid="testnet-transaction-info">
+            {t('[This is %s transaction]').replace('%s', VEGA_ENV)}
```

Comment:
chore (non-blocking):
```suggestion
            {t('[This is a %s transaction only]').replace('%s', VEGA_ENV)}
```

Reply:
ok, but I need also change a test, therefore I push it in other commit